react/rules/ITA.js

```diff
@@ -0,0 +1,72 @@
+import { isPastDate } from '../utils/dateRules'
+
+export default {
+  country: 'ITA',
+  personalFields: [
+    {
+      name: 'firstName',
+      maxLength: 100,
+      label: 'firstName',
+      required: true,
+    },
+    {
+      name: 'lastName',
+      maxLength: 100,
+      label: 'lastName',
+      required: true,
+    },
+    {
+      name: 'email',
+      maxLength: 100,
+      label: 'email',
+      hidden: true,
+    },
+    {
+      name: 'document',
+      maxLength: 50,
+      label: 'document',
+      required: false,
+      hidden: true,
+    },
+    {
+      name: 'homePhone',
+      maxLength: 30,
+      label: 'homePhone',
+    },
```

Comment:
No phone validation?

Reply:
Nops, I'm justing using the default validations and changed the one for document